### PR TITLE
Issue 11

### PR DESCRIPTION
**Pull request by msuhovec**
#11

### CODE REVIEW

File: lib/rof/filters/label.rb [hunk 1]
```diff
@@ -53,6 +54,8 @@ def process(obj_list)
             force = (k == "rels-ext")
             obj[k] = replace_labels(v, labels, force)
           end
+	  master_pid  = obj["pid"] if master_pid .nil?
```
Comment (dbrower) [inline lib/rof/filters/label.rb]: need a space between `master_pid` and `.nil?`

File: lib/rof/filters/label.rb [hunk 2]
```diff
@@ -83,6 +86,17 @@ def replace_labels(obj, labels, force=false)
...
+      def add_bendo_id(obj, master_pid)
+	if obj["bendo-item"]
+	  if  obj["bendo-item"] == ""
+	    obj["bendo-item"]= master_pid.gsub(/^.*:/,"")
+          end
+	end
+
+      	obj
+      end
+
```
Comment (dbrower) [inline lib/rof/filters/label.rb]: Actually, lets assign anything not having a bendo-item a bendo-item.

Comment (msuhovec) [inline lib/rof/filters/label.rb]: Done.

**Comment by dbrower** [comment]
Looks fine as soon as all the tests pass.

**Pull request merged**